### DISCUSSION TITLE
Bubble Menu

### DISCUSSION THREAD

**Discussion by sherminatorrr** [sibiraj-s/ngx-tiptap#69] [General]
Hello together, 

when I use a bubble menu and edit some text with the bubble menu, and use the undo/redo buttons later, the respective past bubble menus that I used before are going to show up again in the editor at the respective position, in my opinion that shouldn't be the case because it's very confusing, the past bubble menus shouldn't show up at all. Is there a way to prevent this behaviour?

Best regards

**Comment by sibiraj-s** [comment]
Hi, I think this behavior not an issue with this package. this package just provides wrappers, behaviors are controlled by TipTap itself. you might want to open up an issue there to get this sorted.
